crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -119,7 +119,17 @@ impl Plugin for MeshRenderPlugin {
                 .add_systems(
                     Render,
                     (
-                        prepare_mesh_uniforms.in_set(RenderSet::PrepareResources),
+                        (
+                            batch_render_phase::<Opaque3dPrepass, MeshPipeline>,
+                            batch_render_phase::<AlphaMask3dPrepass, MeshPipeline>,
+                            batch_render_phase::<Opaque3d, MeshPipeline>,
+                            batch_render_phase::<Transparent3d, MeshPipeline>,
+                            batch_render_phase::<AlphaMask3d, MeshPipeline>,
+                            batch_render_phase::<Shadow, MeshPipeline>,
```

Comment:
@robtfm : It feels like prepass and shadows should be added elsewhere?

Reply:
On closer inspection - shadows are still intertwined here, and the other place for prepass would have been in the prepass plugin, but that is generic over materials and we don't want to add the batching system once per material. So, this is fine.

Reply:
i believe there's a section of the prepass plugin build fn that only runs once - i guess we should move the prepass phases there. 

ultimately when shadows are modularized/pluginized we can move the shadow batch out as well.